torch/csrc/jit/passes/alias_analysis.h

```diff
@@ -45,6 +45,9 @@ class AliasDb {
   bool writesToAlias(Node* n, const ValueSet& vs, bool recurseBlocks = false)
       const;
 
+  // Does `container` hold in memory any element that exist in elem
+  bool mayContainAlias(const Value* elem, const Value* container) const;
```

Comment:
This implementation is not commutative but the one in MemoryDAG is. This is potentially pretty confusing, and I see no blocker to making both implementations commutative

Reply:
i guess i would just call the current function with the order of the arguments reversed